public/src/components/Navbar.js

```diff
@@ -1,27 +1,61 @@
 import styled from 'styled-components';
 import { Link } from 'react-router-dom';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 
 export const Navbar = styled.nav`
     padding: .25em 1em;
     display: flex;
     align-items: center;
+    
 `;
 export const NavbarItemList = styled.ul`
     list-style: none;
     display: flex;
-    
+    margin-top: 0;
     padding-left: 0;
     margin-bottom: 0;
 `;
 export const NavbarItem = styled.li`
     padding: .25em 1em;
+    display: flex;
+    align-items: center;
+
+`;
+export const StyledIcon = styled(FontAwesomeIcon);
```

Comment:
You'll want to do ``; here. styled(FontAwesomeIcon) returns a function

Reply:
Fixed!